math/mathcore/src/MixMaxEngineImpl.h

```diff
@@ -8,16 +8,20 @@
 #define ROOT_Math_MixMaxEngineImpl
 
 
-#if (_N==17)
+#if (_NN==17)
```

Comment:
Macro named starting with underscore are technically reserved for/by the compiler.  Could we use ROOT_MM_N ? (ROOT MixMax N)

Reply:
Anything would be fine with me, but I suppose one should ask the author of the code ( or @lmoneta )